src/casync.h

```diff
@@ -152,11 +152,15 @@ int ca_sync_get_hardlink_digest(CaSync *s, CaChunkID *ret);
 
 int ca_sync_get_seed_requests(CaSync *s, uint64_t *ret);
 int ca_sync_get_seed_request_bytes(CaSync *s, uint64_t *ret);
+int ca_sync_get_seed_time(CaSync *s, uint64_t *ret);
```

Comment:
btw, I think it would be nice to specify the unit in the funciton name, hence `ca_sync_get_seed_time_nsec()`. We do that already for the `bytes` function above

Reply:
agreed

Reply:
Done for this function and others, ie. `ca_sync_get_decoding_time_nsec()`, `ca_sync_get_runtime()`